app/layout.styl

```diff
@@ -27,7 +27,8 @@ form
   padding-bottom 60px
 
   // TODO: Make responsive
-  width 1170px
+  //width 1170px
+  width: 100%
```

Comment:
You are right, its more a general improvement. I just added it because loader was unadjusted when state is isWaiting.